laravel-app/resources/views/emails/auth/email/verification_email.blade.php

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        .email-content {
+            background-color: #f4f4f4;
+            padding: 20px;
+            font-family: Arial, sans-serif;
+        }
+        
+        .button {
+            background-color: #007BFF;
+            color: #fff;
+            text-decoration: none;
+            padding: 10px 20px;
+            border-radius: 5px;
+            display: inline-block;
+        }
+
+    </style>
```

Comment:
Create seperate css file

Reply:
I tried creating css file but somehow the email wont render it
This is also similar to the stackoverflow threads I saw
https://stackoverflow.com/questions/14085630/why-css-is-not-working-when-sending-html-email

Although using `inline styles` works sir and this is the most recommended
https://customer.io/blog/how-to-make-css-play-nice-in-html-emails-without-breaking-everything/

Reply:
Gmail doesn't recognize some styles like the style for the background and button after creating a file. 
for reference:
https://www.emailonacid.com/blog/article/email-development/12_things_you_must_know_when_developing_for_gmail_and_gmail_mobile_apps-2/
Is it okay to use `inline styles` instead of creating css file?